### PR TITLE
Enhancement: Adds `response` method to allow sending plain responses

### LINKED ISSUE #40: Support Plain Symfony Response.
In case i use the Symfony Serializer and call `->json(...)` it will be encoded twice. I could not find any possibility to send a plain request with my own content. For example:

**Double encoded JSON**
```php
// ...
        $entities = $this->repository->findByStuff($stuff);

        return $this->json(
            $this->serializer->serialize($entities, 'json', ['groups' => EntityGroup::READ]),
            Response::HTTP_OK,
            [
                'Content-Type' => 'application/json',
            ],
        );
```

**This works obviously without double encoding**
```php
// ...
        $entities = $this->repository->findByStuff($stuff);

        return new Response(
            $this->serializer->serialize($entities, 'json', ['groups' => EntityGroup::READ]),
            Response::HTTP_OK,
            [
                'Content-Type' => 'application/json',
            ],
        );
```

So it would be great to have a `->response()` method

### PR DESCRIPTION
**Pull request by silasjoisten**
closes #40

### CODE REVIEW

**Comment by OskarStark** [comment]
Friendly ping @azjezz

File: src/Psr7Responder.php [hunk 1]
```diff
@@ -81,6 +81,18 @@ public function route(string $route, array $parameters = [], int $status = Respo
         );
     }
 
+    /**
+     * Returns a Response with given content, status code and headers.
+     *
+     * @param array<string, string> $headers
+     */
+    public function response(string $content = '', int $status = Response::HTTP_FOUND, array $headers = []): ResponseInterface
```
Comment (silasjoisten) [inline src/Psr7Responder.php:89]:
```suggestion
    public function response(?string $content = '', int $status = Response::HTTP_FOUND, array $headers = []): ResponseInterface
```

**Review by azjezz — APPROVED** [review]

**Review by azjezz — COMMENTED** [review]
after seeing #40, i don't think this is needed to solve your problem.

The `json()` method allows you to pass custom context to be used, however, this can be a good edition for creating custom responses.

**Comment by silasjoisten** [comment]
Thanks did not know thats possible! Maybe i used the wrong example here as well. However that would be a great addition

**Pull request merged**